src/components/MeetTheTeam/MeetTheTeam.jsx

```diff
@@ -0,0 +1,212 @@
+import { useEffect, useState } from 'react';
+import { collection, query, getDocs } from 'firebase/firestore';
+import { flushSync } from 'react-dom';
+import {
+    Flex,
+    Box,
+    Container,
+    Title,
+    Text,
+    Avatar,
+    SimpleGrid,
+    createStyles,
+} from '@mantine/core';
+import { FratBoiDug } from '../../assets';
+import { store } from '../../services/firebase';
+
+const MEET_THE_TEAM_TITLE = '<Meet The Team />';
```

Comment:
can change to alias imports '@assets'

services will need  to be added to vite config